werf.yaml

```diff
@@ -726,6 +726,9 @@ import:
   add: /images_tags.json
   to: /deckhouse/modules/images_tags.json
   after: setup
+shell:
+  setup:
+    - ln -s /deckhouse/modules/images_tags.json /deckhouse/modules/040-node-manager/images_tags.json
```

Comment:
It seems we can just import images_tags.json one more time in the `import` section.